tools/make/diplomat-coverage/src/allowlist.rs

```diff
@@ -338,6 +338,7 @@ lazy_static::lazy_static! {
         "icu::calendar::any_calendar::AnyCalendarKind",
         "icu::casemap::titlecase::TitlecaseMapper",
         "icu::calendar::types::Time",
+        "icu::locale::DataLocale",
```

Comment:
I think "reexported" is the wrong section. You probably want to rust_link it to `ffi::Locale`.